content/articles/top-container-orchestration-tools/index.md

```diff
@@ -0,0 +1,137 @@
+Kubernetes automates deployment, scaling, and management of containerized applications.
+
+![Kubernetes Architecture](/engineering-education/top-container-orchestration-tools/kubernetes-architecture.jpg)
+
+[Image source](https://sensu.io/blog/how-kubernetes-works)
+
+Kubernetes architecture consists of four major components.
+**Node** - It handles the running of tasks in a container. Nodes serve as hosts for container runtimes and ease communication between containers.
+**Cluster** - Composed of nodes that handle sharing of resources and manage containerized applications.
+**Replication Controllers** - They are responsible for scheduling and resource allocation on containers
+**Pods** - A scheduling unit that allocates resources to containers 
+
+##### Features of Kubernetes
+- Provides powerful scheduling features.
+- It is secure with Google's Network policies
+- Easy to deploy and manage applications.
+- enhanced portability
+- Enable container isolation by removing interdependencies
+
+#### Google Kubernetes Engine (GKE)
+[GKE](https://cloud.google.com/kubernetes-engine) is a cloud orchestration tool hosted on the Google Cloud Platform. Kubernetes power GKE clusters. It has all Kubernetes functionalities. These functionalities include deployment, scaling, administration, and management of containerized applications.
+
+GKE enables developers to smoothen operations based on application requirements. It provides CI/CD tools to help DevOps improve productivity while building and managing containers.
+
+##### Features of GKE 
+- GKE enables developers to streamline operations based on application needs
+- Provides automated deployment, scaling, and upgrading.
+- Enhance portability
+- Google improves the GKE platform and make it robust and reliable
+
+#### Amazon Elastic Container Service (ECS)
+[Amazon ECS](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ECS_instances.html) is an orchestration tool that runs container clusters on [Amazon EC2 instances](https://aws.amazon.com/ec2/instance-types/). ECS powers services such as Amazon Sage Maker, Amazon Batch, and Amazon recommendation engine.
+
+Amazon ECS provides a reliable, scalable, and secure orchestration platform. Therefore Amazon ECS can be considered appropriate for critical and sensitive applications.
+
+##### Features of ECS
+- Run and manage containers
+- Efficient load balancing
+- Container Registry make applications compatible within different environments
+- ECS integrates with other AWS services to display greater capabilities
+
+#### Azure Kubernetes Service (AKS)
+[AKS](https://azure.microsoft.com/en-us/services/kubernetes-service/) is an open-source container orchestration tool. It is a managed Kubernetes service offered by Azure. It facilitates deploying, scaling, and managing container applications in a cluster environment. Kubernetes master and nodes are configured automatically by AKS. Thus it becomes easy to run tasks in containers.
```

Comment:
Just a suggestion.

I could see that this point `deploying, scaling, and managing container applications in a cluster environment` is repeated more than 2 - 3 times previously.

You have already mentioned this while defining the functionality of Orchestration, so inherently these tools would perform the same functionalities. You need not repeat them for each tool.

Instead, you could compare these tools (performance, scalability) or mention the products that currently use their services or add external links that give more info about these tools.